protocol/testing/containertest/testnet_test.go

```diff
@@ -80,6 +84,9 @@ func expectProto(t *testing.T, tag string, message proto.Message) bool {
 }
 
 func TestPlaceOrder(t *testing.T) {
+	if os.Getenv("SKIP_DISABLED") != "" {
+		t.Skip("Skipping disabled test")
```

Comment:
Would it be helpful to log out test name being skipped? Or does the test output already make that clear